README.md

```diff
@@ -107,6 +107,7 @@ The following options are supported.  See [values.yaml](/charts/atlantis/values.
 | `terminationGracePeriodSeconds`             | Set terminationGracePeriodSeconds for the StatefulSet. | `{}` |
 | `statefulSet.securityContext`               | Allow customizing fsGroup/runAsUser. | `{}` |
 | `statefulSet.priorityClassName`             | Leverage a PriorityClass to ensure your pods survive resource shortages. | `{}` |
+| `statefulSet.updateStrategy`                | Configure [updateStrategy](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies) of the StatefulSet.                            | n/a |
```

Comment:
Shouldn't the `n/a` be `{}` (and surrounded by backticks) so it matches `statefulSet.labels` ?

Reply:
Sure, updated to \`{}\`.